### PR TITLE
add printing css

### LINKED ISSUE #1095: add printing css
When printing a event detail page, the layout is all over the place

### PR DESCRIPTION
**Pull request by felixrindt**
fix #1095

### CODE REVIEW

**Comment by jeriox** [comment]
nice one!

maybe we can retain the gray background for the participant labels so that they have more contrast to the white background? and the disposition button could also use no-print

image

**Comment by felixrindt** [comment]
> nice one!
> 
> maybe we can retain the gray background for the participant labels so that they have more contrast to the white background? and the disposition button could also use no-print
> 
> image
> 

That depends on the "print backgrounds" option in the print dialogue.

I do think we should look into breakpoints to avoid orphaning or windowing the shift box.

File: ephios/core/templates/core/fragments/shift_box_big.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
         </div>
     </div>
 
-    <div class="card-footer">
+    <div class="card-footer no-print">
```
Comment (jeriox) [inline ephios/core/templates/core/fragments/shift_box_big.html:22]: add for disposition button as well

**Review by jeriox — APPROVED** [review]

File: ephios/core/templates/core/fragments/shift_box_big.html (hunk 1 above)
Comment (felixrindt) [reply ephios/core/templates/core/fragments/shift_box_big.html:22]: Thought about moving that to the card-footer.

**Pull request merged**